AMS MathSciNet.js

```diff
@@ -161,97 +161,12 @@ var testCases = [
 	},
 	{
 		"type": "web",
-		"url": "http://www.ams.org/mathscinet/search/publications.html?pg1=ISSI&s1=308850",
+		"url": "https://mathscinet.ams.org/mathscinet/search/publications.html?pg1=ISSI&s1=308850",
 		"items": "multiple"
 	},
 	{
 		"type": "web",
-		"url": "http://www.ams.org.libezproxy2.syr.edu/mathscinet-getitem?mr=2767535",
```

Comment:
I wouldn't delete this test completely but rather change the url to a non-proxified url, i.e. https://mathscinet.ams.org/mathscinet-getitem?mr=2767535 . Otherwise this looks good.